disnake/ui/input_text.py

```diff
@@ -23,22 +23,20 @@
 """
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, Optional, Type, TypeVar
+from typing import TYPE_CHECKING, Optional, Tuple, TypeVar
 
 from ..components import InputText as InputTextComponent
-from ..enums import ComponentType
+from ..enums import ComponentType, InputTextStyle
+from .item import WrappedComponent
 
 if TYPE_CHECKING:
-    from ..enums import InputTextStyle
     from ..types.components import InputText as InputTextPayload
 
-IT = TypeVar("IT", bound="InputText")
-
 
 __all__ = ("InputText",)
 
 
-class InputText:
+class InputText(WrappedComponent):
     """Represents a UI input text.
 
     This can only be used in :class:`~.ui.Modal`.
```

Comment:
Can we move `style` below `custom_id`? Basically to match the `__init__`
![image](https://user-images.githubusercontent.com/67214928/151229755-16fa3ac9-5430-40d2-b708-48b78d08aeb2.png)